tutorials/Transfer_Learning_with_Megatron_for_Languages_Other_than_English.ipynb

```diff
@@ -0,0 +1,893 @@
+        "BRANCH = 'main'\n",
+        "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[nlp]"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "pycharm": {
+          "name": "#%%\n"
+        },
+        "id": "pC0slAc0h9zN"
+      },
+      "source": [
+        "# If you're not using Colab, you might need to upgrade jupyter notebook to avoid the following error:\n",
+        "# 'ImportError: IProgress not found. Please update jupyter and ipywidgets.'\n",
+        "\n",
+        "! pip install ipywidgets\n",
+        "! jupyter nbextension enable --py widgetsnbextension\n",
+        "\n",
+        "# Please restart the kernel after running this cell"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "dzqD2WDFOIN-"
+      },
+      "source": [
+        "from nemo.collections import nlp as nemo_nlp\n",
+        "from nemo.utils.exp_manager import exp_manager\n",
+        "\n",
+        "import os\n",
+        "import wget \n",
+        "import torch\n",
+        "import pytorch_lightning as pl\n",
+        "from omegaconf import OmegaConf\n",
+        "\n",
+        "import zipfile\n",
+        "import random"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "daYw_Xll2ZR9"
+      },
+      "source": [
+        "# Task Description\n",
+        "**Transfer Learning** is a machine learning technique to use an already trained model on a task to be reused on a second task as a starting point. In this tutorial, we will show how NVIDIA's Megatron and Google's BERT language models that are trained on English language can be used for other languages, e.g., Persian language. The downstream task that we will be using for this purpose will be Named entity recognition (NER). NER is the task of detecting and classifying key information (entities) in text.\n",
+        "For example, in a sentence:  `Mary lives in Santa Clara and works at NVIDIA`, we should detect that `Mary` is a person, `Santa Clara` is a location and `NVIDIA` is a company.\n",
+        "\n",
+        "For more information on transfer learning, please see the following:\n",
+        "https://ruder.io/transfer-learning/\n",
+        "https://ieeexplore.ieee.org/abstract/document/5288526\n",
+        "\n",
+        "In this tutorial we will be using Megatron and BERT language models. You can have access to papers of these models using the following: https://arxiv.org/abs/1909.08053, https://arxiv.org/abs/1810.04805\n",
+        "\n",
+        "To read more about other topics and downstream task that can be done in NeMo, you can see the NeMo's tutorial page here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/v1.0.2/starthere/tutorials.html\n"
+      ]
```

Comment:
You mean instead of 
!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[nlp]
I use this:
!python -m pip install git+https://github.com/NVIDIA/NeMo.git@${BRANCH}#egg=nemo_toolkit[nlp]

Reply:
no, I'm talking about this line you can see the NeMo's tutorial page here: `https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/v1.0.2/starthere/tutorials.html\n`
It contains hardcoded NeMo version

Reply:
https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/{BRANCH}/starthere/tutorials.html

Doesn't work.

Reply:
try `https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/tutorials.html`